### ISSUE TITLE
Performance Question

### ISSUE THREAD

**Issue by Gitoffthelawn** [wildskyf/TextareaCache#109]
I use a few websites that are slow due to their large number of HTML elements, large number of complex CSS rulesets, extensive use of JavaScript, and frequent XHR requests.

Should adding these sites to TextareaCache's `exception list` result in any performance improvement?

I tried doing this for one of the sites, and didn't notice any difference right away.  Before I run a few performance metrics, I thought it best to just ask: *Should adding sites to TextareaCache's `exception list` result in any performance improvement on those sites?*

**Comment by GHolk** [comment]
Adding a website to the exception list means that TextareaCache will not run on the site,
so will not affect the performance.

I have no idea if it was still slow after add the website to the exception list (and reload the existing page).

**Comment by Gitoffthelawn** [comment]
> Adding a website to the exception list means that TextareaCache will not run on the site, so will not affect the performance.
> 
> I have no idea if it was still slow after add the website to the exception list (and reload the existing page).

Sorry, let me rephrase.  When sites are slow (unrelated to TextareaCache), it is generally best to not have any additional code (from web extensions) running in them, so as to not slow them down even more.  Thus, I'm wondering if adding such sites to Textarea Cache's exception list is generally a good idea. (Of course, no textarea content will then be cached for such sites.)
